web-common/src/features/dashboards/time-series/MetricsTimeSeriesCharts.svelte

```diff
@@ -372,7 +372,6 @@
           <MeasureBigNumber
             {measure}
             value={bigNum}
-            isMeasureExpanded={isInTimeDimensionView}
```

Comment:
We need this prop for the "Big Number" component because the component is a button in the explore screen and a div in the Time dimension detail screen. It's also styled a bit differently (tile vs plain). Removing this also removes that behavior.

Reply:
Ah, I noticed the style changes when the prop is removed. Passing isInTimeDimensionView to BigMeasureNumber causes the tooltip to flash briefly during transitions from Overview to Measure view. Ideally, we should disable the tooltip during these page transitions.

I'll explore other ways to remove the tooltip flashes and keep the div/button style.

Reply:
Addressed

https://github.com/user-attachments/assets/15252cab-7256-4224-865a-6f17ca4e4c0d